bbcdll/bbc_tcp_server.py

```
@@ -372,28 +380,15 @@ def get_config():
-        "blue": "blue",
-        "copper": "copper",
-        "colorful": "colorful"
-    }
-
-    BATTLE_TYPE_MAP = {
-        "continuous": 0,
-        "tower": 1,
-        "连续出击": 0,
-        "自动编队爬塔": 1
-    }
+    VALID_APPLE_TYPES = {"gold", "silver", "blue", "copper", "colorful"}
```

Ruff (0.15.11)

[warning] Mutable default value for class attribute

(RUF012)